src/dashboards/Compact/index.tsx

```diff
@@ -100,6 +115,62 @@ const EnturDashboard = ({ history }: Props): JSX.Element => {
 
     const maxWidthCols = COLS[breakpoint]
 
+    const prevNumberOfStopPlaces = usePrevious(numberOfStopPlaces)
+
+    const [modalVisible, setModalVisible] = React.useState(false)
+
+    useEffect(() => {
+        const defaultTileOrder: TileItem[] = []
+        if (stopPlacesWithDepartures) {
+            if (stopPlacesWithDepartures.length == prevNumberOfStopPlaces) {
+                return
+            }
+            stopPlacesWithDepartures.map((item) => {
+                defaultTileOrder.push({ id: item.id, name: item.name })
+            })
+        }
+        if (anyBikeRentalStations) {
+            defaultTileOrder.push({ id: 'city-bike', name: 'Bysykkel' })
+        }
+        if (mapCol) {
+            defaultTileOrder.push({ id: 'map', name: 'Kart' })
+        }
+        setTileOrder(defaultTileOrder)
+    }, [
+        stopPlacesWithDepartures,
+        prevNumberOfStopPlaces,
+        anyBikeRentalStations,
+        mapCol,
+    ])
+
+    const reorder = (
+        list: TileItem[],
+        startIndex: number,
+        endIndex: number,
+    ) => {
+        const result = Array.from(list)
+        const [removed] = result.splice(startIndex, 1)
```

Comment:
`splice` bør man unngå, for den likner veldig på den meir brukte `slice`-funksjonen. `splice` muterer, men ikkje `slice`. Derfor er det fint om man held seg kun til `slice`. 
Her kan man sikkert bruke andre metodar også for å få til det man vil.

Reply:
Kanskje noko sånt?
```
const itemToMove = list[startIndex]
let result = list.filter((_, index) => index !== startIndex)
result = [
    ...result.slice(0, endIndex),
    itemToMove,
    ...result.slice(endIndex + 1)
]
return result
```